Kinvey/Kinvey/RealmSync.swift

```diff
@@ -56,6 +56,12 @@ class RealmSync<T: Persistable>: SyncType where T: NSObject {
         log.verbose("Saving pending operation: \(pendingOperation)")
         executor.executeAndWait {
             try! self.realm.write {
+                if !pendingOperation.collectionName.isEmpty,
```

Comment:
@heyzooi this makes sense. When you push, how many pending operations are retrieved at one time to push? Is the the entire queue, or is there a batch size?